content/en/blog/2019/opentelemetry-governance-committee-explained/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: OpenTelemetry Governance Committee Explained
+linkTitle: Governance Committee Explained
+date: 2019-11-06
+canonical_url: https://medium.com/opentelemetry/opentelemetry-governance-committee-explained-860353baba0
+author: "[Sergey Kanzhelev](https://github.com/SergeyKanzhelev)"
+---
+
+This article describes the functions and responsibilities of the OpenTelemetry Governance Committee, based on the charter document found [here](https://github.com/open-telemetry/community/blob/master/governance-charter.md). It is an opinion, not the formal definition. The primary role of the Governance Committee is not to centralize power, but to enable and empower the broader community by establishing processes. Let me explain.
```

Comment:
I thought we had to break at 80? I prefer this style, to be honest, and we might want to revisit the recommendation from here: https://github.com/open-telemetry/opentelemetry.io/blob/main/archetypes/blog.md#paragraphs

Reply:
Yes, we break at 80, I forgot to run the linter, we can discuss changing that separately, not sure if there's a technical reason why it has to be 80.